librz/analysis/arch/ppc/ppc_il.c

```diff
@@ -530,6 +589,47 @@ RZ_IPI ut32 ppc_fmx_to_mask(const ut8 fmx) {
 		(fmx & 0x01 ? x : 0));
 }
 
+static const char *get_crx_reg(const csh handle, cs_insn *insn, size_t n) {
+#if CS_API_MAJOR == 5 && CS_API_MINOR == 0
+	// bug on crx not being populated in capstone v5.0
```

Comment:
If there is an issue open about this please link it.

Reply:
https://github.com/capstone-engine/capstone/issues/2087